moveit_setup_assistant/package.xml

```diff
@@ -21,11 +21,13 @@
   <build_depend>moveit_ros_planning</build_depend>
   <build_depend>moveit_ros_visualization</build_depend>
   <build_depend>yaml-cpp</build_depend>
+  <build_depend version_gt="0.3.1">srdfdom</build_depend>
  
   <run_depend>moveit_core</run_depend>
   <run_depend>moveit_ros_planning</run_depend>
   <run_depend>moveit_ros_visualization</run_depend>
   <run_depend>yaml-cpp</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend version_gt="0.3.1">srdfdom</run_depend>
```

Comment:
> Maybe using a version_gte="0.3.2" would be clearer?

+0.5. I had the same confusion as @rhaschke. But I guess the version of the depended package (srdfdom in this case) is *technically* out of control from this repo, so I guess the current `version_gt="0.3.1"` is the safest.